### ISSUE TITLE
[backend] Don't let the user use an alternative login method based on the same email

### ISSUE THREAD

**Issue by pfortuna** [Tech4covid19/trackovid19-backend#151]
When a user logs in with e.g. Google and is using a specific email e.g. foo@gmail.com, if we previously had a registration using another login method with the same email, we should reject that registration and an error should be returned that asks the user to use the same login method he used before.

**Comment by pfortuna** [comment]
@geiras We need design of the error message /dialog.

**Comment by geiras** [comment]
Would be a possibility to remove the button of the wrong sign-in method after the warning message?

We already have design for error messages. Can we use the same, @pfortuna?

**Comment by pfortuna** [comment]
> Would be a possibility to remove the button of the wrong sign-in method after the warning message?
Yes this is a possibility. And once we know, we also set a cookie to remember the correct login method. But sometimes we don't know that until we check the backend and return an error.

> We already have design for error messages. Can we use the same, @pfortuna?
yes we can :-) can you post it here for facilitating?

**Comment by geiras** [comment]
I've designed new errors for this situation. 
![image](https://user-images.githubusercontent.com/22840903/78510137-f7837080-778a-11ea-90ad-b5feaa518977.png)

https://www.figma.com/file/BfK7HtPsvxZAwZLg1ebz2u/Covidografia?node-id=4007%3A7059
https://www.figma.com/file/BfK7HtPsvxZAwZLg1ebz2u/Covidografia?node-id=4007%3A7121

**Comment by pfortuna** [comment]
Perfect ! moving this to TODO!

**Comment by geiras** [comment]
This way the user can login with the correct method in the same popup where we show the error.
